src/measure_theory/measure/measure_space.lean

```diff
@@ -1566,6 +1566,11 @@ lemma is_finite_measure_of_le (μ : measure α) [is_finite_measure μ] (h : ν 
   is_finite_measure ν :=
 { measure_univ_lt_top := lt_of_le_of_lt (h set.univ measurable_set.univ) (measure_lt_top _ _) }
 
+lemma measure.is_finite_measure_map [measurable_space α] -- why do we need this here but not above?
```

Comment:
In the lemma above, the statement involves `ν`, which puts that measure as well as its measurable space into the context.
```suggestion
lemma measure.is_finite_measure_map {m : measurable_space α}
```
As a rule, make the measurable space argument implicit if it can be inferred from the measure. Otherwise you will need to use `@` if your measure is a trim (and hence defined on a non-default measurable space).